server/routes/api.js

```diff
@@ -0,0 +1,69 @@
+const express = require('express')
+const router = express.Router()
+const moment = require('moment')
+const Expense = require('./model/Expense')
+
+const getDate = (isExist) => (isExist ? isExist.format('LLLL') : moment().format('LLLL'))
```

Comment:
instead of calling it "isExist", it's preferable to call it "date", because you expect to get a date there (or not, and then create a date yourself, as you are doing)